apps/server/src/modules/account/uc/account.uc.spec.ts

```diff
@@ -848,16 +865,6 @@ describe('AccountUc', () => {
 				await expect(accountUc.searchAccounts(currentUser, params)).rejects.toThrow(ForbiddenOperationError);
 				configSpy.mockRestore();
 			});
-
-			it('teacher can not access any account of a foreign school via user id', async () => {
```

Comment:
why was this case removed?

Reply:
That was a duplicated test case, probably an artifact from another merge.